tests/test_sudoku_board.cc

```diff
@@ -0,0 +1,169 @@
+#include "test_sudoku_board.h"
+
+void TestSudokuBoard::print_board_values(){
```

Comment:
```suggestion
void TestSudokuBoard::print_board_values() {
```
You more places like this